src/components/page-header/page-header.component.spec.ts

```diff
@@ -87,4 +100,16 @@ describe('Page Header Component', () => {
         expect(pageHeader.textContent).toContain('UX');
     });
 
+    it('should add href to breadcrumbs with a routerLink', () => {
+        const homeBreadcrumb = nativeElement.querySelector('.breadcrumb a');
+        expect(homeBreadcrumb.textContent.trim()).toBe('Home');
+        expect(homeBreadcrumb.hasAttribute('href')).toBe(true);
+    });
+
+    it('should not add href to breadcrumbs without a routerLink', () => {
```

Comment:
The acceptance criteria mentions a new test page, but I think these karma tests achieve the same result and are probably faster, so I'll stick with this unless there's a specific need for an e2e test that I'm not aware of yet

Reply:
I think that having both test coverage could be a good idea because while I do think the tests present are good I do see the use case for having a new test page currently to my knowledge we do not have any e2e tests that mention the routerLink so it would be good to have some coverage for this.

Reply:
I'm not sure if duplicating the karma tests into e2e tests provides much value though. The karma tests cover the fixed behaviour and duplicating tests would possibly be a bit overkill I think

Reply:
That's true it might be overkill for the bug but if it is in the acceptance criteria I think it should be implemented

Reply:
It's not specifically useful for this fix, but it would be good coverage infill to verify that routerlinks work as expected when specified, which I don't think we can cover with Karma.

Reply:
@donna-marijne what would be the best way of verifying the routerLinks work? In the test above I've used `hasAttribute('href')` to check for the presence of `href`. Would possibly using `getAttribute('href')` and asserting on the value of the `href` be sufficient without needing an e2e test?

Reply:
I don't know what the href would look like within a karma test, or whether we could depend on that value through updates to Angular or Karma. Since the router is an external component we don't really want to be unit testing it. What we are interested in is the outcome i.e. that the expected content is rendered in the router outlet after clicking an item with a routerlink, or that nothing happens if a routerlink is not provided.

Reply:
I've added some e2e tests to cover the navigation 👍